Lib/test/test_descr.py

```diff
@@ -1347,6 +1347,8 @@ class Meta(type):
             def __new__(cls, name, bases, namespace, attr):
                 self.assertIn(attr, namespace)
                 return super().__new__(cls, name, bases, namespace)
+            def __init__(cls, name, bases, namespace, attr):
+                super().__init__(name, bases, namespace)
```

Comment:
add `__init__` to prevent `attr` from being passed through to `__init_subclass__`